src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -76,17 +95,26 @@ resource dataFactory 'Microsoft.DataFactory/factories@2018-06-01' existing = {
   name: dataFactoryName
 }
 
+// Get storage account instance
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' existing = {
+  name: storageAccountName
+}
+
+// Get keyvault instance
+resource keyVault 'Microsoft.KeyVault/vaults@2023-02-01' existing = {
+  name: keyVaultName
+}
+
 //------------------------------------------------------------------------------
-// Stop all triggers before deploying
+// Identities and RBAC
 //------------------------------------------------------------------------------
 
 // Create managed identity to start/stop triggers
 resource identity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
-  name: '${dataFactoryName}_${exportContainerName}_extract_triggerManager'
+  name: '${dataFactory.name}_${exportContainerName}_extract_triggerManager'
```

Should this be generic for all triggers? Or are you wanting to leave it as-is given it's already been created?
```suggestion
  name: '${dataFactory.name}_triggerManager'
```